templates/gosu/game.rb

```diff
@@ -0,0 +1,22 @@
+require 'gosu'
```

Comment:
should we consider a more topically neutral file name for `gosu` apps? Maybe `app.rb`?